Cubical/Data/Int/Properties.agda

```diff
@@ -451,6 +827,311 @@ inj-+z {z} {l} {n} p = inj-z+ {z = z} (+Comm z l ∙ p ∙ +Comm n z)
 n+z≡z→n≡0 : ∀ n z → n + z ≡ z → n ≡ 0
 n+z≡z→n≡0 n z p = inj-z+ {z = z} {l = n} {n = 0} (+Comm z n ∙ p)
 
+pos+posLposMin : ∀ x y → min (pos (x +ℕ y)) (pos x) ≡ pos x
```

Comment:
In the naming of these, should Min/Max come first?

Reply:
When this is refactored per our discussion, this lemma won't exist anymore.

Reply:
Great!